daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector.md

```diff
@@ -66,11 +63,7 @@ That's it! There's no need include any SDKs or instrument your application code.
 
 > **NOTE**: You can register multiple tracing exporters at the same time, and the tracing logs are forwarded to all registered exporters.
 
-Deploy and run some applications. After a few minutes, you should see tracing logs appearing in your Application Insights resource. You can also use **Application Map** to examine the topology of your services, as shown below:
-
-![Application map](/images/open-telemetry-app-insights.png)
```

Comment:
Any reason you removed the image of showing an app map in AI?

Reply:
This file shows instructions not related to AppInsights, so I removed.